package.json

```diff
@@ -12,16 +12,21 @@
     "coverage": "jest --coverage"
   },
   "dependencies": {
-    "bootstrap": "^5.0.1",
+    "@babel/runtime-corejs3": "^7.14.7",
+    "@types/jquery": "2.0.47",
+    "axios": "^0.21.1",
+    "bootstrap": "^5.0.2",
     "file-loader": "^6.2.0",
```

Comment:
@DianaVargasOrellana a similar comment for it: if we are not using this library, we should remove it from dependencies.